### DISCUSSION TITLE
Wert wird nicht angezeigt...

### DISCUSSION THREAD

**Discussion by tbit-c** [inventwo/ioBroker.vis-inventwo#434] [Q&A] [answered]
Hallo,

ich versuche nun schon seit geraumer Zeit, einfach nur einen Wert der Homematic Wetterstation anzuzeigen. Es funktioniert aber nicht. So schaut im IOBroker aus:
<img width="1602" alt="Bildschirmfoto 2022-04-11 um 13 28 12" src="https://user-images.githubusercontent.com/69671358/162730555-5e4c40e9-0e34-4775-83ee-982f838c6f54.png">
Im VIS schaut es so aus:
<img width="724" alt="Bildschirmfoto 2022-04-11 um 13 39 34" src="https://user-images.githubusercontent.com/69671358/162732171-4b0109cf-904e-478a-b727-39d7a5b90781.png">
Habt Ihr einen Tip? Mache ich was falsch?
Dank und Gruß
Tom

**Comment by skvarel** [comment, marked as answer]
Das klappt so nicht. Du musst den Datenpunkt inkl. geschweifter Klammern (Binding) in das Beschriftungsfeld eintragen.

{hm-rp.1........} km/h

Dann wird der Wert angezeigt.
